documentation/src/docs/pravega-concepts.md

```diff
@@ -11,362 +11,266 @@ You may obtain a copy of the License at
 Pravega.
 
-A Writer is an application that creates Events and writes them into a Stream.
-All data is written by appending to the tail (front) of a Stream.
-
-A Reader is an application that reads Events from a Stream.  Readers can read
-from any point in the Stream.  Many Readers will be reading Events from the tail
-of the Stream.  These Events will be delivered to Readers as quickly as possible.  Some Readers will read
-from earlier parts of the Stream (called catch-up reads).  The application
-developer has control over where in the Stream the Reader starts reading.
- Pravega has the concept of a Position, that represents where in a Stream a
-Reader is currently located.  The Position object can be used as a recovery
-mechanism – applications that persist the last Position a Reader has
-successfully processed can use that information to initialize a replacement
-Reader to pickup where a failed Reader left off.  Using this pattern of
-persisting Position objects, applications can be built that guarantee exactly
-once Event processing in the face of Reader failure.
-
-Readers are organized into ReaderGroups.  A ReaderGroup is a named collection of
-Readers that together, in parallel, read Events from a given Stream. When a
-Reader is created through the Pravega data plane API, the developer includes the
-name of the ReaderGroup it is part of.  We guarantee that each Event published
-to a Stream is sent to exactly one Reader within the ReaderGroup.  There could
-be 1 Reader in the ReaderGroup, there could be many.  There could be many
-different ReaderGroups simultaneously reading from any given Stream.
-
-You can think of a ReaderGroup as a "composite Reader" or a "distributed
-Reader", that allows a distributed application to read and process Stream data
-in parallel, such that a massive amount of Stream data can be consumed by a
-coordinated fleet of Readers in a ReaderGroup.  A collection of Flink tasks
-processing Stream data in parallel is a good example use of a ReaderGroup.
-
-For more details on the basics of working with Pravega Readers and Writers,
-see [Working with Pravega: Basic Reader and
-Writer](basic-reader-and-writer.md#working-with-pravega-basic-reader-and-writer).
+- **Writer:** An application that creates events and writes them into a stream.
+All data is written by appending to the tail (front) of a stream.
+
+- **Reader:** An application that reads events from a stream.  Readers can read
+from any point in the stream.  Many readers will be reading events from the tail
+of the Stream. The tail reads correspond to reading bytes that have been recently written and will be delivered to readers immediately. Some readers will read from earlier parts or history of the stream (called **catch-up reads** or **history reads**). The application developer has control over the reader's start position in the stream.
+
+- **Position:** A concept in Pravega, that represents where in a stream a
```

Comment:
For the first sentence, I suggest: `**Position:** Abstraction that represents where in a Stream a reader is currently located.